app/src/main/java/co/kr/deartoday/presentation/ui/signin/SignInActivity.kt

```diff
@@ -18,15 +19,16 @@ class SignInActivity : BaseActivity<ActivitySignInBinding>() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        signIn()
+        setOnLayoutButtonClickListener()
     }
 
-    private fun signIn() {
+    private fun setOnLayoutButtonClickListener() {
```

Comment:
길지만 명확한 함수 네이밍 좋네요 ^^

Reply:
저는 다른 네이밍이 더 좋을 것 같아요.

Reply:
```suggestion
    private func setButtonListenerWhoSettingOnLayout() {
```

근데 도대체 리스너가 뭔데요? ㅋㅋ 일단 suggest.